docs/info/terms-and-conditions.md

```diff
@@ -0,0 +1,290 @@
+and understood Agreements, and agree to be bound of them.
+
+If you do not agree with (or cannot comply with) Agreements, then you may not use the Service, but please let us know by
+emailing at info@expresslrs.org so we can try to find a solution. These Terms apply to all visitors, users and others
+who wish to access or use Service.
+
+## 2. Communications
+
+By using our Service, you agree to subscribe to newsletters, marketing or promotional materials and other information we
+may send. However, you may opt out of receiving any, or all, of these communications from us by following the
+unsubscribe link or by emailing at info@expresslrs.org.
+
+## 3. Contests, Sweepstakes and Promotions
+
+Any contests, sweepstakes or other promotions (collectively, “Promotions”) made available through Service may be
+governed by rules that are separate from these Terms of Service. If you participate in any Promotions, please review the
+applicable rules as well as our Privacy Policy. If the rules for a Promotion conflict with these Terms of Service,
+Promotion rules will apply.
+
+## 4. Content
+
+Our Service allows you to post, link, store, share and otherwise make available certain information, text, graphics,
+videos, or other material (“Content”). You are responsible for Content that you post on or through Service, including
+its legality, reliability, and appropriateness.
+
+By posting Content on or through Service, You represent and warrant that: (i) Content is yours (you own it) and/or you
+have the right to use it and the right to grant us the rights and license as provided in these Terms, and (ii) that the
+posting of your Content on or through Service does not violate the privacy rights, publicity rights, copyrights,
+contract rights or any other rights of any person or entity. We reserve the right to terminate the account of anyone
+found to be infringing on a copyright.
+
+You retain any and all of your rights to any Content you submit, post or display on or through Service and you are
+responsible for protecting those rights. We take no responsibility and assume no liability for Content you or any third
+party posts on or through Service. However, by posting Content using Service you grant us the right and license to use,
+modify, publicly perform, publicly display, reproduce, and distribute such Content on and through Service. You agree
+that this license includes the right for us to make your Content available to other users of Service, who may also use
+your Content subject to these Terms.
+
+ExpressLRS has the right but not the obligation to monitor and edit all Content provided by users.
+
+In addition, Content found on or through this Service are the property of ExpressLRS or used with permission. You may
+not distribute, modify, transmit, reuse, download, repost, copy, or use said Content, whether in whole or in part, for
+commercial purposes or for personal gain, without express advance written permission from us.
```

Comment:
"without express advance written permission from us."
I don't think this applies, so might as well put in "without sufficient attribution to the source material."